app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(list_of_elves: list) -> None:
+    for elf in list_of_elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
+    for dwarf in dwarves:
+        dwarf.eat_favourite_dish()
```

Comment:
Please use similar naming for declared lists
`ilst_of_elves` change to `elves` or `dwarves` change to `list_of_dwarves`